packages/dapp/src/components/MultiSigConfirmationModal/MultiSigConfirmationModal.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Modal, Typography } from '@mui/material';
+import {
+  MultisigConfirmationModalContainer,
+  MultisigConfirmationModalIcon,
+  MultisigConfirmationModalIconContainer,
+} from './MultisigConfirmationModal.styles';
+import { ButtonPrimary } from '@transferto/shared';
+import { useTranslation } from 'react-i18next';
+
+export const MultisigConfirmationModal: React.FC<{
+  open: boolean;
+  onClose: () => void;
+}> = ({ open, onClose }) => {
+  const i18Path = 'multisig.transactionInitiated';
+
+  const { t: translate } = useTranslation();
+
+  return (
+    <Modal open={open} onClose={null}>
```

Comment:
Shouldn't we pass `onClose` param here as well?

Reply:
I thought we want the user to acknowledge the message about it being a SAFE wallet, hence making the modal closable by clicking on the CTA rather than clicking outside and closing. wdyt?

Reply:
Not sure, I don't like this technic 😄